go/runtime/config/config.go

```diff
@@ -279,10 +282,19 @@ type PruneConfig struct {
 	NumKept uint64 `yaml:"num_kept"`
 }
 
+// IndexerConfig is history indexer configuration.
+type IndexerConfig struct {
+	// BatchSize is max number of blocks committed in a batch during history reindex.
+	//
+	// Setting it to zero (default), sets default batch size.
```

Comment:
```suggestion
	// Setting it to zero uses the default batch size.
```
Confusing, as one might think that zero is default.
Good APIs would probably also expose the default value, so that users know what the default value is.